README.md

```diff
@@ -446,6 +446,38 @@ Validates included structures. Must supply the name of a valid include.
 Examples:
 - `include('person')`
 
+### Field Names in Constraints
+Constraints can now access the field name they're validating, enabling more descriptive error messages. This is particularly useful for custom constraints that need to provide field-specific messages.
```

Comment:
Consider changing to "Constraints can access the field name they're validating, enabling descriptive error messages". Removes the words "now" and "more". 

Since this is a README, the text should indicate the feature has always been there....since this text will be there for the next 10 years. :)